index.bs

```diff
@@ -204,14 +204,18 @@ NOTE: The dimensions indicated in the 'ispe' property might not match the values
 
 NOTE: The values of [=render_width_minus1=] and [=render_height_minus1=] possibly present in the AV1 bistream are not exposed in the AVIF container level.
 
+<h4 id="clean-aperture-property">Clean Aperture Property</h4>
+
+The semantics of the clean aperture property ('clap') as defined in [[!HEIF]] apply. In addition to the restrictions on transformative item property ordering specified in [[!MIAF]], the following restriction also applies:
+
+<assert>The origin of the 'clap' item property shall be anchored to 0,0 (top-left) unless the full, un-cropped image item is included as a secondary item in an image collection.</assert>
```

Comment:
As said during the call, I would have liked stronger constraints such as dimensions but this is too complicated.

Reply:
I confirm that Chrome requires the clean aperture to be anchored at (0, 0) (top left) of the coded input image, and does not impose other requirements on the clean aperture.

We can modify Chrome to require that less than 64 pixels are cropped from the width or height of the coded input image. This will support the use case of supporting encoders that require width or height to be a multiple of 64. After gaining experience of this constraint in Chrome, we can then add it to the AVIF spec.

Reply:
@wantehchang 
> We can modify Chrome to require that less than 64 pixels are cropped from the width or height of the coded input image. This will support encoders that require width or height to be a multiple of 64. After gaining experience with this constraint in Chrome, we can then add it to the AVIF spec.

I understand the want for having stronger constraints, but we've had stuff like that come back and bite us multiple times in the past. We currently know of no AV1 HW that requires larger alignment than 64, but for HEVC we have had HW that requires horizontal alignments of:
- 64
- 128
- 192
- 256

And vertical alignments of:
- 64
- 96
- 128

Assuming that all future AV1 HW will stick to 64x64 sounds like a dangerous assumption to me.

-----------------

Threat:
The main thing we want to protect against is some kind of editing service that promises to do lossless cropping of your edit and that simply applies a `clap` to the file **without the user knowing that the full image is still present**. 

Valid use cases we want to be able to support:
- Dealing with HW limitations
- Useful tool for more advanced capture concepts

From my memory of the discussion, everyone could agree on two mitigations that would more or less completely mitigate the threat while still supporting the valid use-cases:
- Limit the `clap` to be anchored to the top-left corner. (This makes it useless as a general purpose cropping tool)

unless
- The full un-cropped image is also present and visible in the file. (This ensures that if used for generic cropping, the user is made aware that the content is still present.)